spec/abci/abci.md

```diff
@@ -303,7 +303,7 @@ via light client.
   - `ByzantineValidators ([]Evidence)`: List of evidence of
     validators that acted maliciously.
 - **Response**:
-  - `Tags ([]kv.Pair)`: Key-Value tags for filtering and indexing
+  - `Events ([]abci.Event)`: Key and multiple events for filtering.
```

Comment:
Now that we have an Event type we should explicitly document it here. Also need to add some docs for the new `.index` field. Should be made clear that this is a non-deterministic subjective field that can be different on different nodes.